test/test_models.py

```diff
@@ -594,6 +594,14 @@ def test_vitc_models(model_fn, dev):
     test_classification_model(model_fn, dev)
 
 
+@pytest.mark.parametrize(
+    "model_fn", [models.max_vit_T_224, models.max_vit_S_224, models.max_vit_B_224, models.max_vit_L_224]
+)
+@pytest.mark.parametrize("dev", cpu_and_gpu())
+def test_max_vit(model_fn, dev):
+    test_classification_model(model_fn, dev)
```

Comment:
We shouldn't do this. If you register the model it with the registration mechanism your test will be automatically picked up and run on the `test_classification_model()`

Reply:
I've used that test as a way of solely testing the `maxvit` configurations. Is there any way to specify a subset of models on when running `test_classification_model`?

Reply:
The recommended way is to keep only 1 variant while you develop the main implementation. Not only the tests will be faster but also you reduce the LOC and boilerplate code of the PR, making it easier for others to review.